pkg/daemon/daemon.go

```diff
@@ -1076,7 +1086,9 @@ func (dn *Daemon) applySSHAccessedAnnotation() error {
 func (dn *Daemon) onConfigDrift(err error) {
 	dn.nodeWriter.Eventf(corev1.EventTypeWarning, "ConfigDriftDetected", err.Error())
 	glog.Error(err)
-	dn.updateErrorState(err)
+	if err := dn.updateErrorState(err); err != nil {
+		glog.Infof("Could not update annotation: %v", err)
```

Comment:
In this particular case, use `glog.Error` instead of `glog.Info`. This will append the word "error" next to the log entry which can aid debugging and log monitoring tooling. Do something like this instead:

```golang
if err := dn.updateErrorState(err); err != nil {
  glog.Error(fmt.Errorf("could not update annotation: %w", err))
}
```

Reply:
glog.Error does not cause execution to stop right?

Reply:
`glog.Error()` (and `glog.Errorf()`) will log to stderr and keep going, whereas `glog.Fatal()` (and `glog.Fatalf()`) will log to stderr and call `os.Exit()`. By comparison, `glog.Info()` (and its analogues) will always log to stdout. I hope this clarifies things a bit!